$NetBSD: patch-aa,v 1.2 1999/08/10 11:16:59 agc Exp $

Adjust for gcc, and don't use Sun compiler on Solaris.

--- Makefile.orig	Tue Aug 10 11:59:27 1999
+++ Makefile	Tue Aug 10 12:04:40 1999
@@ -13,7 +13,7 @@
 # defaults:
 PGM = lame
 CC = gcc
-CC_OPTS = -O 	
+CC_OPTS = -O3 
 GTK = 
 GTKLIBS = 
 SNDLIB = 
@@ -98,10 +98,10 @@
 ##########################################################################
 # SunOS
 ##########################################################################
-ifeq ($(UNAME),SunOS) 
-   CC = cc
-   MAKEDEP = -xM
-endif
+#ifeq ($(UNAME),SunOS) 
+#   CC = cc
+#   MAKEDEP = -xM
+#endif
 
 
 ##########################################################################
